test/fw/ptl/utils/pbs_dshutils.py

```diff
@@ -1946,30 +1947,48 @@ def mkdtemp(self, hostname=None, suffix='', prefix='PtlPbs', dir=None,
                      directory
         :param level: logging level, defaults to INFOCLI2
         """
+        # create a temp dir as current user
+        tmpdir = tempfile.mkdtemp(suffix, prefix, "/tmp")
```

Comment:
mkdtemp by default creates dir in system tmp dir. So no need to pass '/tmp'  arg to tempfile.mkdtemp() method